CreateDefaultRoles.md

```diff
@@ -0,0 +1,39 @@
+# HOW TO DEFINE SuperUser #
+
+```php
+    <?php
+    // Create a User:
+    
+    $user = new User();
+    $user->setUsername('{yourusername}');
+    $user->setPassword('{yourpassword}');
+    // Any other data can be set for $user
+    
+    $user = RcmUserService->createUser(User $requestUser, false);
+    
+    // Create Role super admin role:
+    
+    $superAdminRoleId = AclDataService->getSuperAdminRoleId()
```

Comment:
"AclDataService->getSuperAdminRoleId()" returns a response object which can't be piped directoy to setRoleId()